torch/fx/experimental/proxy_tensor.py

```diff
@@ -407,7 +413,7 @@ def inner_torch_dispatch(self, func_overload, types, args=(), kwargs=None):
             # constant, so we keep a copy of the original argument along so
             # we can query it if we're asked to item() it at some later point
```

Comment:
Do we need to handle https://github.com/pytorch/pytorch/blob/1dfba0120490e1b101d3f2d4acbf81ffc0fb475f/torch/fx/experimental/proxy_tensor.py#L348 as well ?